src/pages/Trending.jsx

```diff
@@ -0,0 +1,140 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import '../styles/movielist.css'
+import Ig from '../../public/instagram.svg'
+import Git from '../../public/github.svg'
+import Lk from '../../public/linkedin.svg'
+import Logo from '../../public/logo.png'
+
+function Trending() {
+  const [list, setList]= useState([])
+  const [load, setLoad]= useState(true)
+  const [time, setTime]= useState('day')
+  const [expand, setExpand]= useState(0)
+  const [clicked, setClicked]= useState(false)
+
+  function getExpanded(id) {
+
+    if(clicked === true) {
+      setClicked(false)
+      setExpand(0)
+    }
+    else {
+      setClicked(true)
+      setExpand(id)
+    }
+  }
+
+  function getType(show) {
+    if(show.media_type === 'movie') return show.title
+    else return show.name
+  }
+
+  function getRelease(show) {
+    if(show.media_type === 'movie') return show.release_date
+    else return show.first_air_date
+  }
+
+  useEffect(() => {
+    axios.get(`https://api.themoviedb.org/3/trending/all/${time}?api_key=9e89dc1810e00461d6f59011e04175ed`).then((movies) => {
+      setList(movies.data.results)
+      setTimeout(() => setLoad(false), 1000)
+    })
+  }, [time])
+
+  return (
+    <>
+    {load ? (
+      <div className="container-svg">
+        <svg className='svg' viewBox='0 0 50 50'>
+          <circle
+            cx={25}
+            cy={25}
+            r={20}
+          />
+        </svg>
+      </div>
+    ) 
+    : 
+    (
+      <>
```

Comment:
I think it's better if you do it this way
```suggestion
if(load) {
  return (
      <div className="container-svg">
        <svg className='svg' viewBox='0 0 50 50'>
          <circle
            cx={25}
            cy={25}
            r={20}
          />
        </svg>
      </div>
)
}

return  (
<>
```